fluidBuild.config.cjs

```diff
@@ -270,6 +270,60 @@ module.exports = {
 				"package.json",
 			],
 			"npm-package-json-script-dep": ["^build-tools/"],
+			"npm-public-package-requirements": [
```

Comment:
This is in the handler exclusions section; is that intentional?

Reply:
If it's intentional, then the entries need to be regexes.

Reply:
Oh, interesting. Somehow this worked though... either that or I just encountered some unrelated flub incremental build issue... So these should be file paths or file path regex patterns then?

Reply:
I forgot to re-link and re-build after my most recent changes, or I would have caught this 🤦‍♂️. Will update and validate.